doc/py_tutorials/py_feature2d/py_fast/py_fast.markdown

```diff
@@ -108,18 +108,18 @@ kp = fast.detect(img,None)
 img2 = cv2.drawKeypoints(img, kp, None, color=(255,0,0))
 
 # Print all default params
-print "Threshold: ", fast.getThreshold()
-print "nonmaxSuppression: ", fast.getNonmaxSuppression()
-print "neighborhood: ", fast.getType()
-print "Total Keypoints with nonmaxSuppression: ", len(kp)
+print( "Threshold: ", fast.getThreshold() )
```

Comment:
Or you can use Formatter: `print('Threshold: {}'.format(fast.getThreshold()))` without importing print function.